packages/react-components/src/components/ParticipantItem.tsx

```diff
@@ -206,13 +206,10 @@ export const ParticipantItem = (props: ParticipantItemProps): JSX.Element => {
         title={strings.menuTitle}
         data-ui-id={ids.participantItemMenuButton}
       >
-        <Icon
-          iconName={itemHovered ? 'ParticipantItemOptionsHovered' : 'ParticipantItemOptions'}
```

Comment:
Do we need to remove the declarations of `ParticipantItemOptions` icon now that it is no longer being used?